src/wallet/wallet.cpp

```diff
@@ -1841,9 +1845,20 @@ CWallet::ScanResult CWallet::ScanForWalletTransactions(const uint256& start_bloc
             WalletLogPrintf("Still rescanning. At block %d. Progress=%f\n", block_height, progress_current);
         }
 
+        bool skip_block{false};
```

Comment:
In 73964bc1590f0ba1113ecda7895744bd5f870ee5 "wallet: take use of `FastWalletRescanFilter`"

nit: Since only `skip_block`'s negation is checked, this could perhaps be clearer as `bool fetch_block{true}`, so that we are checking whether a block should be fetched and scanned, rather than whether a block should not be skipped.